Streaming/FFmpegDecoder.cpp

```diff
@@ -261,8 +247,8 @@ namespace moonlight_xbox_dx {
 		decoder_callbacks_sdl.start = startCallback;
 		decoder_callbacks_sdl.stop = stopCallback;
 		decoder_callbacks_sdl.cleanup = cleanupCallback;
-		decoder_callbacks_sdl.submitDecodeUnit = submitCallback;
-		decoder_callbacks_sdl.capabilities = CAPABILITY_DIRECT_SUBMIT | CAPABILITY_REFERENCE_FRAME_INVALIDATION_AVC | CAPABILITY_REFERENCE_FRAME_INVALIDATION_HEVC;
+		decoder_callbacks_sdl.submitDecodeUnit = NULL;
+		decoder_callbacks_sdl.capabilities = CAPABILITY_REFERENCE_FRAME_INVALIDATION_AVC | CAPABILITY_REFERENCE_FRAME_INVALIDATION_HEVC | CAPABILITY_PULL_RENDERER;
```

Comment:
Are you sure that the decoder can actually handle reference frame invalidation for H.264 and HEVC?

It doesn't work for me with D3D11VA on Windows 11.